src/peft/tuners/lora/config.py

```diff
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import annotations
```

Comment:
For my better understanding, do we need to remove this import or should we just avoid using `X | Y` specifically?

Reply:
Without this import, `X | Y` will not work under Python 3.10 as this feature is only introduced in Python 3.10. `from __future__ import annotations` backports this features to lower version of Python, but it breaks `transformers` argument parser.

Reply:
I understand what the import does, my question was if we can leave the import and just avoid `X | Y` or if the mere fact that we have the import already breaks the argument parser.

Reply:
I reckon `hf_argparser` should be augmented to transform `X | Y` to `Union[X, Y]` (a simple regexp!)...